src/mod_vcard.erl

```diff
@@ -134,12 +134,17 @@ get_personal_data(Username, Server) ->
     LServer = jid:nameprep(Server),
     Jid = jid:to_binary({LUser, LServer}),
     Schema = ["jid", "vcard"],
-    Entries = case mod_vcard_backend:get_vcard(LUser, LServer) of
-        {ok, Record} ->
-            SerializedRecords = exml:to_binary(Record),
-            [{Jid, SerializedRecords}];
-         _ -> []
-        end,
+    Entries = lists:flatmap(fun(B) ->
+                                    try B:get_vcard(LUser, LServer) of
+                                        {ok, Record} ->
+                                            SerializedRecords = exml:to_binary(Record),
+                                            [{Jid, SerializedRecords}];
+                                        _ -> []
+                                    catch
+                                        _:_ ->
```

Comment:
Silent error - is there really a reason not to log it? This will be a nightmare to debug.

Reply:
I thought about the same thing after merging and decided to fix it after all other PRs are merged.